### PR TITLE
Update App Icon to Proper Appearance

### PR DESCRIPTION
**Pull request by cedricbahirwe**
- Update the app icon to fit in the Mac Dock

### CODE REVIEW

File: Exercism.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -602,7 +602,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Exercism/Preview Content\"";
-				DEVELOPMENT_TEAM = S82GZQ59NS;
+				DEVELOPMENT_TEAM = P6333LR9H2;
```
Comment (cedricbahirwe) [inline Exercism.xcodeproj/project.pbxproj:605]: Any way to avoid this change of dev team ?

Comment (angiemugo) [reply Exercism.xcodeproj/project.pbxproj:605]: yes @cedricbahirwe , we actually need a `. gitignore`. I am going to add that shortly.

File: Exercism/Util/ExercismApp.swift [hunk 2]
```diff
@@ -41,6 +41,14 @@ struct ExercismApp: App {
                                                                        NSApplication.AboutPanelOptionKey(rawValue: "Copyright"): "© 2023"
                     ] as [NSApplication.AboutPanelOptionKey : Any])
                 }
+
```
Comment (angiemugo) [inline Exercism/Util/ExercismApp.swift:44]: great catch

**Review by angiemugo — APPROVED** [review]
Thanks for this @cedricbahirwe, I'll go ahead and approve it. I will add a .gitignore on another branch

**Pull request merged**